Add task solution

eduvinagre opened the pull request:
- [DEMO LINK](https://eduvinagre.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://eduvinagre.github.io/layout_search-bar-airbnb/report/html_report/)
- [x] Icon implemented using background-image CSS property
- [x] Inputs are written inside of 'form' tag with correctly passed attributes
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

jv-aquino commented on src/index.html:
```diff
@@ -10,23 +10,40 @@
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="big-searchbar"
```
There's a better way to name it, you already did it with the input, if you want I can describe it more but I think you can figure it out - by the way there's also a BEM problem there

eduvinagre replied:
Indeed. I'll update them!

jv-aquino commented on src/index.html:
```diff
@@ -10,23 +10,40 @@
+      data-qa="big"
+    >
+      <input
+        class="input input--big"
```
Remember - the input is part of the Form Block
https://en.bem.info/methodology/quick-start/#membership

eduvinagre replied:
Alright, I tried using the best of the BEM methodology. I hope I've got it right this time.

jv-aquino commented on src/style.css:
```diff
@@ -1 +1,74 @@
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
```
Nice order of declarations / selectors!

jv-aquino commented on src/index.html:
```diff
@@ -10,23 +10,40 @@
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```
You should also add a class to target this form with the special "sauce" that you used in the input

jv-aquino commented on src/style.css:
```diff
@@ -1 +1,74 @@
+
+  background-size: 11px 11px;
+  background-position: 13px center;
+  font-size: 14px;
```
Nice use of the "sauce" in the naming convention and also good spacing

jv-aquino reviewed (changes requested):
Your code is good overall specially the css and some naming convention, but there are things you forgot to apply - check BEM - that will make your code awesome!

eduvinagre commented:
Alright, I tried using the best of the BEM methodology. I hope I've got it right this time. But I think it looks kind of weird. Let me know, please!

EdPirro commented on src/style.css:
```diff
@@ -1 +1,74 @@
+
+.search-form__input--large:focus,
+.search-form__input--small:focus {
+  font-family: Avenir-Heavy, sans-serif;
```
Font family is usually set in the `body` tag to guarantee the entire content will have the same font by default!

eduvinagre replied:
@EdPirro I've gotten a bit confused on this review, as there are 3 different fonts for each action on the search-bars. It changes on focus an hover, that's why I did it this way.

EdPirro commented on src/style.css:
```diff
@@ -1 +1,74 @@
+.search-form__input--large:focus,
+.search-form__input--small:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
```
Your focused input's text seems to be missing a text shadow style as in the Figma Design; could you come up with a way to implement it

Yours:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/96e07d2b-5012-4b2a-904d-2d2e8b4fa0ad)

Figma:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/51e65e04-344b-41a5-9c63-9f28fd455d2d)

eduvinagre replied:
Man, your eagle eyes are absolutely right. I confess I had noticed the shadow, but there were only specifications for box-shadow in Figma, not for text-shadow. I'll take the opportunity and create the shadow for the input of the small search bar.

EdPirro reviewed (changes requested):
Awesome job, @eduvinagre! I just pointed out a few nitpick improvements that you could implement to make this solution simply perfect! 🚀

eduvinagre commented:
So 3/3 l'll hope everything is alright now!

I've inserted a label tag to better deal with the image and also thinking about SEO, and not a <div>. Also, made text shadow changes even though not explicit in the Figma file. Added some new hover features like keeping placeholder with no shadows on the small search-bar. Ran all tests and it looks good.

IMPORTANT: I've did a new deploy but my updated style.css file doesn't seem to be updated at all. I've tried deploying it 4 times, so I'm attaching a video of the features I've introduced to prove it works.

https://github.com/mate-academy/layout_search-bar-airbnb/assets/122939534/7817d9ab-600e-4761-98c7-9aac624e96e6

eduvinagre commented:
@jv-aquino consegue fechar o meu review nesta pull request? Acho que o sistema da Mate não está reconhecendo o novo fork por conta disso.
